service/src/main/java/aero/minova/cas/service/ViewService.java

```diff
@@ -105,163 +122,26 @@ public Table executeView(Table inputTable, List<Row> authoritiesForThisTable) th
-							.collect(Collectors.toList()));
-			while (sqlSet.next()) {
-				outputTable.addRow(SqlUtils.convertSqlResultToRow(outputTable, sqlSet, customLogger.getUserLogger(), this));
-			}
-			return outputTable;
-		} catch (Throwable e) {
-			throw new RuntimeException(e);
-		}
-	}
-
-	public String prepareViewString(Table params, boolean autoLike, int maxRows, List<Row> authorities) throws IllegalArgumentException {
-		return prepareViewString(params, autoLike, maxRows, false, authorities);
-	}
-
-	/**
-	 * Diese Methode stammt ursprünglich aus "ch.minova.ncore.data.sql.SQLTools#prepareViewString". Bereitet einen View-String vor und berücksichtigt eine evtl.
-	 * angegebene Maximalanzahl Ergebnisse
-	 *
-	 * @param params
-	 *            Suchzeilen (z.B. Suchparameter), wobei auch ein Spezialfeld mit dem Namen 'AND' genutzt werden kann, um die Kriterien zu verknüpfen
-	 * @param autoLike
-	 *            wenn true, dann werden alle String-Parameter, die noch kein % haben, mit einem '%' am Ende versehen
-	 * @param maxRows
-	 *            maximale Anzahl Ergebnisse (Zeilen), die die Abfrage liefern soll, 0 für unbegrenzt
-	 * @param count
-	 *            Gibt an ob nur die Anzahl der Ergebniss (Zeilen), gezählt werden sollen.
-	 * @return Präparierter View-String, der ausgeführt werden kann
-	 * @throws IllegalArgumentException
-	 * @author wild
-	 */
-	public String prepareViewString(Table params, boolean autoLike, int maxRows, boolean count, List<Row> authorities) throws IllegalArgumentException {
-		final StringBuffer sb = new StringBuffer();
-		if (params.getName() == null || params.getName().trim().length() == 0) {
-			throw new IllegalArgumentException("msg.ViewNullName");
-		}
-
-		if (count) {
-			sb.append("select count(1) from ");
-		} else {
-			if (maxRows > 0) {
-				sb.append("select top ").append(maxRows).append(" ");
-			} else {
-				sb.append("select ");
-			}
-			val outputFormat = params.getColumns().stream()//
-					.filter(c -> !Objects.equals(c.getName(), Column.AND_FIELD_NAME))//
-					.collect(Collectors.toList());
-			if (outputFormat.isEmpty()) {
-				sb.append("* from ");
-			} else {
-				sb.append(//
-						outputFormat.stream()//
-								.map(Column::getName)//
-								.collect(Collectors.joining(", ")));
-				sb.append(" from ");
-			}
-		}
-		sb.append(params.getName());
-		boolean whereClauseExists = false;
-		if (!params.getColumns().isEmpty() && !params.getRows().isEmpty()) {
-			final String where = prepareWhereClause(params, autoLike);
-			sb.append(where);
-			if (!where.trim().equals("")) {
-				whereClauseExists = true;
-				sb.append(")");
-			}
-		}
-
-		final String onlyAuthorizedRows = securityService.rowLevelSecurity(whereClauseExists, authorities);
-		sb.append(onlyAuthorizedRows);
-
-		return sb.toString();
+		return SqlUtils.convertSqlResultToTable(inputTable, sqlSet, customLogger.getUserLogger(), this);
 	}
 
 	/*
```

Comment:
Müsse pagingWithSeek nicht Teil der ViewServiceInterface sein?
Da es, glaube ich, nicht aktiv genutzt wird, kann man bei der Implementierung in JOOQViewService auch erstmal einfach eine Ausnahme schmeißen.